configuration/config-txt/overclocking.md

```diff
@@ -10,30 +10,61 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
-| temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius.  The default value is `85`. Values over 85 are clamped to 85.|
-| temp_soft_limit | **3A+/3B+ only**. CPU speed throttle control. This sets the temperature at which the CPU clock speed throttling system activates. At this temperature, the clock speed is reduced from 1400Mhz to 1200Mhz.  Defaults to `60`, can be raised to a maximum of `70`, but this may cause instability.
+| initial_turbo | Enables turbo mode from boot for the given value in seconds, or until cpufreq sets a frequency. For more information [see here](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=6201&start=425#p180099). The maximum value is `60`. |
+| arm_freq_min | Minimum value of arm_freq used for dynamic frequency clocking. |
+| core_freq_min | Minimum value of `core_freq` used for dynamic frequency clocking. On Pi 4B the default is`275` when `hdmi_enable_4kp60` is set|
+| gpu_freq_min | Minimum value of `gpu_freq` used for dynamic frequency clocking.|
+| h264_freq_min | Minimum value of `h264_freq` used for dynamic frequency clocking.|
+| isp_freq_min | Minimum value of `isp_freq` used for dynamic frequency clocking.|
+| v3d_freq_min | Minimum value of `v3d_freq` used for dynamic frequency clocking.|
+| sdram_freq_min | Minimum value of `sdram_freq` used for dynamic frequency clocking.|
+| over_voltage_min | Minimum value of `over_voltage` used for dynamic frequency clocking.|
+| temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius. Values over 85 are clamped to 85.|
+| temp_soft_limit | **3A+/3B+ only**. CPU speed throttle control. This sets the temperature at which the CPU clock speed throttling system activates. At this temperature, the clock speed is reduced from 1400Mhz to 1200Mhz.  Defaults to `60`, can be raised to a maximum of `70`, but this may cause instability. |
+
+This table gives the default values for the options on various Raspberry Pi Models, all frequencies are stated in MHz.
+
+| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4  |
+| ---          | :---:    | :---: | :---: | :----:  | :-----: | :----: | 
+| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 |
+| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550/360 |
+| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
+| isp_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
+| v3d_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
+| hevc_freq    | N/A    | N/A | N/A | N/A   | N/A   | 500/550/360 |
+| sdram_freq   | 450    | 400 | 400 | 450   | 500   | 3200 |
+| arm_freq_min | 700    | 700 | 600 | 600   | 600   | 600  |
+| core_freq_min| 250    | 250 | 250 | 250   | 250   | 250/275 |
+| gpu_freq_min | 250    | 250 | 250 | 250   | 250   | 500  |
```

Comment:
I've just tested on my Pi4B running Raspbian with versions:
```
pi@raspberrypi:~ $ vcgencmd version
Jan 30 2020 13:50:45 
Copyright (c) 2012 Broadcom
version 8f792e011c8a0ea3c5d47e7cc10172cc10b93c09 (clean) (release) (start)
pi@raspberrypi:~ $ uname -a
Linux raspberrypi 4.19.97-v7l+ #1294 SMP Thu Jan 30 13:21:14 GMT 2020 armv7l GNU/Linux
```
and with no overclocking options set in `config.txt` :
```
pi@raspberrypi:~ $ grep _min /boot/config.txt 
pi@raspberrypi:~ $ grep freq /boot/config.txt 
#arm_freq=800
```
And I get these values for `core_freq_min` and `gpu_freq_min` ?
```
pi@raspberrypi:~ $ vcgencmd get_config core_freq_min
core_freq_min=200
pi@raspberrypi:~ $ vcgencmd get_config gpu_freq_min
gpu_freq_min=250
```

Even if these "placeholder values" (?) are incorrect, having `core_freq_min` being 250/275 on a Pi4, whilst having `gpu_freq_min` on a Pi4 being 500, looks strange?